src/content/saas-vs-self-hosted.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: SaaS vs. Self-Hosted
+description: The difference between Okteto's product offerings in SaaS and Self-Hosted
+sidebar_label: SaaS vs. Self-Hosted
+id: saas-vs-self-hosted
+---
+
+Okteto consists of two products: our [CLI](/cloud/okteto-cli) and the [Okteto Platform](#). You can use the Okteto CLI without using the Okteto platform, but there are important limitations with that use case. 
+
+The Okteto Platform is the other half of our product experience that provides the fully managed infrastructure for your development environments. It's what interprets the [Okteto Manifest](/reference/manifest) to spin up a development environment and automates your workflows. 
+
+Some examples of what the Okteto paltform manages are: sleeping/waking resources, incorporating external resources into your development environment, and automatically scaling your development environment as your application's needs grow.
```

Comment:
On scale, what we scale is the cluster infra not the specific dev env.

```suggestion
Some examples of what the Okteto platform manages are: sleeping/waking resources, incorporating external resources into your development environment, and automatically scaling your development environment as your application's needs grow.
```

Reply:
Fixed in latest commit: https://github.com/okteto/docs/pull/367/commits/9630097d1c09d0f5355025f301f365404ed1f400